### PR TITLE
Remove Ultrix, OSF/1, Tru64

### PR DESCRIPTION
**Pull request by richsalz**
## Checklist
Thank you for your contribution. Please answer the following:

- [X] I acknowledge that I am authorized to submit this code under
the terms of the [Apache License](https://www.apache.org/licenses/LICENSE-2.0)
- [ ] All new public APIs are documented
- [ ] All new public functions have tests
- [ ] All new public functions have fuzzing tests

### CODE REVIEW

File: Configure [hunk 1]
```diff
@@ -1061,7 +1061,6 @@ while (@argvcopy)
                         push @{$useradd{LDFLAGS}}, $_;
                         }
                 elsif (/^-rpath$/ or /^-R$/)
```
Comment (botovq) [inline Configure:1063]:
```suggestion
                elsif (/^-R$/)
```
Or is the -rpath still needed for other reasons? If so it should be documented why.

File: CHANGES.md [hunk 2]
```diff
@@ -38,3 +38,4 @@ Currrent Changes
   - Removed WinCE support.
   - Removed IRIX support.
   - Removed `I386_only` support.
+  - OSF/1, True64, and Utrix support.
```
Comment (botovq) [inline CHANGES.md:41]:
```suggestion
  - OSF/1, True64, and Ultrix support.
```

**Review by botovq — CHANGES_REQUESTED** [review]
Looks reasonable. Wonder if this can also go away now:

https://github.com/quictls/quictls/blob/243a6716a0efc7d1740728df91d448539c944ef3/util/shlib_wrap.sh.in#L98-L102

File: Configure (hunk 1 above)
Comment (richsalz) [reply Configure:1063]: I don't know if some other system needs it and the comment was just never updated.  I figure that is more than likely.

**Comment by richsalz** [comment]
I will simplify the exec, but put a note in CHANGES.md about it.

**Comment by richsalz** [comment]
fixup commit pushed

File: Configure (hunk 1 above)
Comment (botovq) [reply Configure:1063]: Oh, it's the -rpath flag that lld also has. Never mind my suggestion then (I should have looked at the context).

File: CHANGES.md [hunk 3]
```diff
@@ -24,6 +24,7 @@ Currrent Changes
   - fips, no-fips since we do not build a FIPS-validated container
   - ecdh, it was a no-op
 
+
```
Comment (botovq) [inline CHANGES.md:27]: What's that extra blank line for? Did you mean to document the exec change here?

**Review by botovq — APPROVED** [review]
I'm fine with it, but I think you wanted to add something to CHANGES.md

**Review by botovq — APPROVED** [review]

**Pull request merged**